hare.spec

```diff
@@ -45,7 +45,7 @@ BuildRequires: python36-devel
 BuildRequires: python36-pip
 BuildRequires: python36-setuptools
 
-Requires: consul = 1.7.8
+Requires: consul >= 1.7.0, consul < 1.10.0
```

Comment:
The last version we tested Hare with was 1.9.1, I think we should restrict the higher limit to the last tested version.

Reply:
They land only fixes when the last number is increased which don't break anything usually. I think it's safe to assume that if it works with 1.9.1 - it will work with any future 1.9.x versions then.

Reply:
Fair enough.